drivers/sensor/bh1749/Kconfig

```diff
@@ -0,0 +1,23 @@
+#
+#  Copyright (c) 2019 Nordic Semiconductor ASA
+#
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig BH1749
+	bool "BH1749 Sensor"
+	depends on I2C && HAS_DTS_I2C && HAS_DTS_GPIO && SENSOR
```

Comment:
`&& SENSOR` is unnecessary here since this file is read from within `if SENSOR`.

Reply:
fixed